### PR TITLE
DP-2798 - Write a Parquet file using schema from database

### PR DESCRIPTION
**Pull request by gvjoshi25**

### CODE REVIEW

File: example/data/input/user_information.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+CREATE TABLE user_role (
+  user_role_id INT NOT NULL,
+  user_role VARCHAR(32) NOT NULL,
+  PRIMARY KEY (user_role_id)
+ );
+  
+CREATE TABLE user_information (
```
Comment (DeleTaylor) [inline example/data/input/user_information.sql]: Can we call this table `user`?

File: example/data/input/user_information.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+INSERT INTO user_role
+(user_role_id, user_role)
+VALUES
+(1, 'Admin'), (2, 'Viewer');
+
+INSERT INTO user_information 
+(user_id, user_name, first_name, last_name, email, password, user_role_id) VALUES
+(1, 'john_doe', 'John', 'Doe', 'john.doe@northconcepts.com', '**********', 1),
+(2, 'jane_doe', 'jane', 'Doe', 'jane.doe@northconcepts.com', '**********', 1),
+(3, 'view_user', 'viewer', '', 'view_user@northconcepts.com', '**********', 2);
```
Comment (DeleTaylor) [inline example/data/input/user_information.sql]: Let's use `@example.com` as the email domain.

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: example/data/input/user_information.sql (hunk 1 above)
Comment (gvjoshi25) [reply example/data/input/user_information.sql]: Done.

File: example/data/input/user_information.sql (hunk 2 above)
Comment (gvjoshi25) [reply example/data/input/user_information.sql]: Done.

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**